src/modules/aws_ecr/index.ts

```diff
@@ -31,16 +31,18 @@ class RepositoryImageMapper extends MapperBase<RepositoryImage> {
     return Object.is(a.registryId, b.registryId);
   };
 
-  async repositoryImageMapper(image: ImageAws, ctx: Context, type: string, region?: string) {
+  async repositoryImageMapper(image: ImageDetail, ctx: Context, type: string, region?: string) {
     const out = new RepositoryImage();
 
     // id is generated with imageDigest + tag
-    if (!image.imageId?.imageDigest) {
+    if (!image.imageDigest) {
       throw new Error('Invalid repository image');
     }
-    out.imageDigest = image.imageId.imageDigest;
-    out.imageTag = image.imageId.imageTag ?? '<untagged>';
-
+    const imageTag = image.imageTags && image.imageTags.length > 0 ? image.imageTags[0] : '<untagged>';
```

Comment:
You may be able to replace this with:

```suggestion
    const imageTag = image?.imageTags?.[0] ?? '<untagged>';
```